modules/run/artifacts/opt/jboss/container/java/run/run-java.sh

```diff
@@ -96,13 +103,8 @@ load_env() {
   if [ -z "${JAVA_APP_DIR}" ]; then
     # XXX: is this correct?  This is defaulted above to /deployments.  Should we
     # define a default to the old /opt/java-run?
-    JAVA_APP_DIR="${JBOSS_CONTAINER_JAVA_RUN_MODULE}"
-  else
-    if [ -f "${JAVA_APP_DIR}/${run_env_sh}" ]; then
```

Comment:
This code used to check for a run-env.sh file in JAVA_APP_DIR, which is *not necessarily* JBOSS_CONTAINER_JAVA_RUN_MODULE, and if it exists, call it. We need to leave this in: it's to support a user providing their own run-env.sh alongside their application.